server/app/schemas.py

```diff
@@ -12,37 +12,68 @@
 
 
 class UserBase(BaseModel):
+    """
+    Base Class for a user
+    """
+
     name: Optional[str]
     role: str = "user"
 
 
 class UserCreate(UserBase):
+    """
+    Class to create a user.
+    Inherits from UserBase class.
+    """
+
     iss: str  # Issuer
     sub: str  # Subject
 
 
 class User(UserBase):
+    """
+    Class for a user.
+    Inherits from UserBase.
+    @params:
+    """
+
     id: int
 
     class Config:
+        """Orm mode configuration"""
+
         orm_mode = True
 
 
 # Auth
 
 
 class AuthorizationResponse(BaseModel):
+    """
+    Base Class for a Authorization Response when
+    using the API
+    """
+
     state: str
     code: str
 
 
 class Token(BaseModel):
+    """
+    Base Class for defining a Token.
+    The token is also linked to a user.
+    """
+
     access_token: str
     token_type: str
     user: User
```

Comment:
This should probably be `UserBase` (client doesn't need to know the `id`). Don't fiddle with this in this PR, it's in bug #43.